README.md

```diff
@@ -1,20 +1,28 @@
-# Smaller immutable `Vec<u*>`s
-This library provides a more efficient way of storing immutable
-lists of unsigned integer values.
+# PackedVec
 
-A `PackedVec` is a `struct` that stores the elements of a given `Vec<u64>` as
-a series of`N` bit values, where `N` is the number of bits needed to represent
-the largest number from the original list.
+A [`PackedVec`](https://docs.rs/packedvec/) stores vectors of integers
+efficiently while providing a API similar to `Vec`. The basic idea is to store
+each element using the minimum number of bits needed to represent every element
+in the `Vec`. For example, if we have a `Vec<u64>` with elements [20, 30, 140],
+every element wastes most of its 64 bits: 7 bits is sufficient to represent the
```

Comment:
Do we want to explain why 7 bits are enough? I think the reason may not be immediately obvious. We could leave this for another PR, because we'll probably need to explain exactly *how* we are compressing the input vector.

Reply:
I think people can work it out themselves easily enough; and, in a sense, it doesn't really matter *how* we compress things, just that we compress things. The full details are also a bit complex with unsigned numbers... They can always read the source code ;)